packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -571,9 +571,12 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
 
     protected renderFileNode(node: SearchInWorkspaceFileNode): React.ReactNode {
         const icon = node.icon;
+        const fileUri = new URI(node.fileUri);
```

Comment:
I don't think there's a need to have a variable since it is only ever being used once.
Directly in the title you can do something similar:

```ts
title={new URI(node.fileUri).path.toString()}
```